README.md

```diff
@@ -45,7 +45,7 @@ Also, you may have to make sure the storage node is in the network
 map.
 
 ``` sh
-$ docker exec -ti sn neofs-cli  control netmap-snapshot -c /config/cli-cfg.yaml --endpoint 127.0.0.1:16513
+$ docker exec -ti sn neofs-cli netmap snapshot -c /config/cli-cfg.yaml --endpoint 127.0.0.1:16513
```

Comment:
It seems we should use:
```
$ docker exec -ti sn neofs-cli netmap snapshot -c /config/cli-cfg.yaml --rpc-endpoint 127.0.0.1:8080
```

Reply:
Sorry wanted and forgot to replace this.